src/chart/Treemap.tsx

```diff
@@ -327,6 +328,11 @@ export interface Props {
 
   style?: React.CSSProperties;
 
+  /**
+   * This is aspect ratio of both the chart, and the individual treemap rectangles.
+   * If you define both width and height on the treemap, this prop will be ignored for the main chart,
+   * but will still be used for the rectangles.
+   */
   aspectRatio?: number;
```

Comment:
Also callout on Treemap.aspectRatio - this prop was there before, but only applied on the individual nodes. I think it makes sense to have it apply on the chart as a whole as well.